osquery/tables/networking/etc_hosts.cpp

```diff
@@ -62,11 +63,22 @@ QueryData parseEtcHostsContent(const std::string& content) {
 
 QueryData genEtcHosts(QueryContext& context) {
   std::string content;
+  QueryData qres = {};
+
   if (readFile(kEtcHosts, content).ok()) {
-    return parseEtcHostsContent(content);
-  } else {
-    return {};
+    qres = parseEtcHostsContent(content);
+  }
+
+#ifdef WIN32
+  content.clear();
```

Comment:
Why clear the content from /etc/hosts? Does the ICS override it completely? Assuming there are unique host entries in both hosts and hosts.ics, do we want both?

If ISC overrides then we only want to replace the collisions right?

Reply:
content is just a temporary string buffer to hold the contents of file read from disk. We parse the bytes and render them into QueryData type.
So we wanna clear it before adding hosts.ics contents.

Reply:
I guess alternatively we could try to read both files into `content` first and then call `parseEtcHostsContent(content)` once, that would work too.